adapters/info.go

```diff
@@ -217,3 +218,7 @@ func parseBidderInfo(info BidderInfo) parsedBidderInfo {
 	}
 	return parsedInfo
 }
+
+type ExtraRequestInfo struct {
+	PbsEntryPoint pbsmetrics.RequestType
+}
```

Comment:
Hi Gus,  The OpenRTB.BidRequest structure is an industry standard independent of PBS that we are not free to change.  While it would ease this particular task if we could update the structure, it wouldn't make sense to have PBS-specific fields in that struct.  See https://www.iab.com/wp-content/uploads/2015/05/OpenRTB_API_Specification_Version_2_3_1.pdf.

Reply:
Also the `openrtb` package is a third party package, so we would need to fork it, and then track any updates in order to make any changes to it.